tests/test_cnn.py

```diff
@@ -109,14 +109,21 @@ def test_feature_extractor_target_net(model_class, share_features_extractor):
         if model_class != QRDQN:
             assert id(model.policy.actor.features_extractor) != id(model.policy.critic.features_extractor)
 
-    # Critic and target should be equal at the begginning of training
+    # Critic and target should be equal at the beginning of training
     params_should_match(model.critic.parameters(), model.critic_target.parameters())
 
     model.learn(200)
 
     # Critic and target should differ
     params_should_differ(model.critic.parameters(), model.critic_target.parameters())
 
+    # Force stats copy
+    model.target_update_interval = 1
+    model._on_step()
+
+    # running stats are copied, see https://github.com/DLR-RM/stable-baselines3/issues/996
+    params_should_match(model.batch_norm_stats, model.batch_norm_stats_target)
```

Comment:
`model.batch_norm_stats` is an empty list here, no?

Reply:
Oh I was thinking since this is after line 115, I'd better force the target copy and try to check. Why is it an empty set?